grub.d/99_migration

```diff
@@ -17,6 +17,11 @@ migration_iso=$(echo /usr/share/migration-image/*-Migration.*.iso)
 root_device=$(lsblk -p -n -r -o NAME,MOUNTPOINT | grep -E "/$" | uniq | cut -f1 -d" ")
 root_uuid=$(blkid -s UUID -o value "${root_device}")
 root_type=$(blkid -s TYPE -o value "${root_device}")
+
+if [[ $root_type =~ ^ext[[:digit:]] ]] ; then
```

Comment:
Just nitpick but for consistency shell variables should be referenced in bracket notation

```bash
if [[ "${root_type}" =~ ^ext[[:digit:]] ]] ; then
```